src/App.jsx

```diff
@@ -14,45 +16,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+function getSortedGoods(listOfGoods, { sortType, isReverse }) {
+  let copyOfGoods = [...listOfGoods];
+
+  copyOfGoods.sort((good1, good2) => {
+    switch (sortType) {
+      case SORT_FIELD_ALPH:
+        return good1.localeCompare(good2);
+      case SORT_FIELD_LENGTH:
+        return good1.length - good2.length;
+      default:
+        return 0;
+    }
+  });
+
+  if (isReverse === true) {
+    copyOfGoods = [...copyOfGoods].reverse();
```

Comment:
```suggestion
    copyOfGoods.reverse();
```
can be more simplify